presto-product-tests/src/main/java/io/prestosql/tests/hive/TestViewfsHiveTable.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.tests.hive;
+
+import com.google.inject.Inject;
+import io.prestosql.tempto.AfterTestWithContext;
+import io.prestosql.tempto.BeforeTestWithContext;
+import io.prestosql.tempto.ProductTest;
+import io.prestosql.tempto.hadoop.hdfs.HdfsClient;
+import org.testng.annotations.Test;
+
+import static io.prestosql.tempto.assertions.QueryAssert.Row.row;
+import static io.prestosql.tempto.assertions.QueryAssert.assertThat;
+import static io.prestosql.tempto.query.QueryExecutor.query;
+import static io.prestosql.tests.TestGroups.VIEWFS;
+import static io.prestosql.tests.utils.QueryExecutors.onHive;
+import static io.prestosql.tests.utils.QueryExecutors.onPresto;
+
+public class TestViewfsHiveTable
+        extends ProductTest
+{
+    @Inject
+    private HdfsClient hdfsClient;
+
+    @BeforeTestWithContext
+    public void setup()
+    {
+        hdfsClient.createDirectory("/user/hive/warehouse/TestViewfsTable/single_column");
+    }
+
+    @AfterTestWithContext
+    public void cleanup()
+    {
+        hdfsClient.delete("/user/hive/warehouse/TestViewfsTable");
+    }
+
+    @Test(groups = VIEWFS)
+    public void testSelectViewfsTable()
+    {
+        onPresto().executeQuery("DROP TABLE IF EXISTS test_select_viewfs");
+        onPresto().executeQuery("" +
+                "CREATE TABLE test_select_viewfs" +
+                " (name VARCHAR) " +
+                "WITH ( " +
+                "   format = 'TEXTFILE', " +
+                "   external_location = 'viewfs://hadoop-viewfs/default/TestViewfsTable/single_column'" +
+                ")");
+        onHive().executeQuery("INSERT INTO test_select_viewfs VALUES ('a'), ('b'), ('c')");
```

Comment:
This is slow. Please populate data with Presto. 

1. create table A with some data with Presto (CTAS) 
2. create external table B over viewfs in Presto, pointing at A's location
3. query B and verify results